src/components/templates/Page.jsx

```diff
@@ -51,13 +52,22 @@ const BackLink = styled.a`
   }
 `
 
+const LogoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
Should be okay. But be careful when overwriting the default values (here  _row_). There is already a lot of logic on how the content is shown. If you change layout attributes, please always test them in many different screen sizes (mobile and desktop) and with different settings of the logo size (at least XS, M and XL).